src/templates/tr/2019/base.html

```diff
@@ -0,0 +1,22 @@
+{% extends "%s/base.html" % lang %}
+
+{% block methodology_stat_1 %}5.8M{% endblock %}
+{% block methodology_stat_2 %}20.9 TB{% endblock %}
+{% block total_websites %}yaklaşık 6 milyon{% endblock %}
+{% block dataset %}Temmuz 2019{% endblock %}
+
+{% block foreword %}
+  <p>
+    Açık web, inanılmaz derecede karmaşık, gelişen bir teknoloji ağıdır. Tüm endüstriler ve kariyerler web üzerine kuruludur ve başarılı olmak için onun canlı ekosistemine bağlıdır. Web eleştirel olduğu kadar, onun nasıl çalıştığını anlamak şaşırtıcı derecede zordur. 2010&#8217;dan beri, HTTP Archive projesinin misyonu, web&#8217;in nasıl inşa edildiğini ve harika bir iş çıkardığını izlemek. Ancak, kapatılması özellikle zor olan bir boşluk var: HTTP Archive projesinin topladığı veriye anlam getirmek ve web&#8217;in nasıl performans gösterdiğinin kolayca anlaşılması için topluluğa imkan sağlamak. Bu Web Almanac&#8217;ın devreye girdiği yerdir.
+  </p>
+
+  <p>
+    Web Almanac&#8217;ın misyonu, aksi takdirde yalnızca cesur veri madencileri tarafından erişilebilecek içgörü hazinesini almak ve anlaşılması kolay bir şekilde paketlemektir. Bu, veriyi anlamlandırabilen ve bize ne anlama geldiğini söyleyebilen endüstri uzmanlarının yardımıyla mümkün olur. Web Almanac&#8217;taki 20 bölümün her biri web&#8217;in belirli bir yönüne odaklanır, ve her bir bölüm kendi alanındaki uzmanlar tarafından yazılmış ve kendi meslekdaşları tarafından gözden geçirilmiştir. Web Almanac&#8217;ın gücü doğrudan onu yazan kişilerin uzmanlığından gelir.
+  </p>
+
+  <p>
+    Bu raporda öğrenilecek çok şey var, öyleyse keşfetmeye başlayın ve öğrendiklerinizi toplulukla paylaşın, böylece web&#8217;in durumuna ilişkin anlayışımızı toplu olarak ilerletebiliriz.
+  </p>
```

Comment:
We missed a line here:

It should be this:

```html
  <p>
    Many of the findings in the Web Almanac are worthy of celebration, but it&#8217;s also an important reminder of the work still required to deliver high-quality user experiences. The data-driven analyses in each chapter are a form of accountability we all share for developing a better web. It&#8217;s not about shaming those that are getting it wrong, but about shining a guiding light on the path of best practices so there is a clear, right way to do things. With the continued help of the web community, we hope to make this an annual tradition, so each year we can track our progress and make course corrections as needed.
  </p>

  <p>
    There is so much to learn in this report, so start exploring and share your takeaways with the community so we can collectively advance our understanding of the state of the web.
  </p>
```

But think you only included the second line. Can you add the first line as well?

Reply:
Second line has been added into 2019/base.html